core-processor/src/processing.rs

```diff
@@ -155,26 +159,45 @@ where
                 process_allowance_exceed(dispatch, program_id, res.gas_amount.burned())
             }
         }),
-        Err(ExecutionError::Actor(e)) => Ok(process_error(
+        Err(ExecutionError::Actor(e)) => Ok(process_execution_error(
             dispatch,
             program_id,
             e.gas_amount.burned(),
             SystemReservationContext::default(),
             e.reason,
-            true,
+            false,
```

Comment:
why false

Reply:
because it was false. I have changed logic in this PR insignificantly. The logic is that if Actor error occurs than it means that it's impossible to instantiate env for execution by some reason, so we just skip signal sending, because it won't be able to be executed also.